tests/page-objects/wallet.page.ts

```diff
@@ -113,7 +113,7 @@ export class WalletPage {
   }
 
   async waitForHomePage() {
-    await this.page.waitForSelector(this.$homePageContainer, { timeout: 30000 });
+    await this.page.waitForSelector(this.$homePageContainer, { timeout: 60000 });
```

Comment:
Is this necessary?

Reply:
I had to add this or the balance test was failing during CI waiting for the home page. Did your changes to the password validation cause any new delays? I can try to run them again today w/out this change, but just couldn't get it to pass yesterday. Reran them 3-4 times.